lib/common/results.c

```diff
@@ -293,6 +293,10 @@ static const struct pcmk__rc_info {
       "Two or more XML elements have the same ID",
       -pcmk_err_generic,
     },
+    { "pcmk_rc_disabled",
+      "Object is disabled",
```

Comment:
I'm thinking just "Disabled". The idea is that the log message has more context than "object" like `"Ignoring alert %s: %s", alert_id, pcmk_rc_str(rc)`

Reply:
I prefer that too. I was afraid you'd say it was too short haha. The current version offers no additional info though.